docker-entrypoint.sh

```diff
@@ -16,6 +16,7 @@ file_env() {
 if [ "$1" = 'node' ]; then
     file_env 'GH_SECRET'
     file_env 'GH_TOKEN'
+		file_env 'SLACK_URL'
```

Comment:
Please fix the indentation
```suggestion
    file_env 'SLACK_URL'
```